src/site/content/en/learn/html/links/index.md

```diff
@@ -181,7 +181,7 @@ or `<label>` either. While the HTML page will still render, nesting focusable an
 
 ## Links and JavaScript
 
-The `links` attribute returns an `HTMLCollection` matching `a` and `area` elements that have an `href` attribute.
+The `links` property returns an `HTMLCollection` matching `a` and `area` elements that have an `href` attribute.
```

Comment:
Does this seem like the correct nomenclature, @rachelandrew? I think it is from a JavaScript perspective, but not entirely sure since there's overlap with HTML concepts here cc: @estelle

Reply:
yeah that looks correct.